digits/model/tasks/caffe_train.py

```diff
@@ -411,7 +411,10 @@ def save_files_classification(self):
                     val_data_layer.hdf5_data_param.batch_size = constants.DEFAULT_BATCH_SIZE
 
         # hidden layers
-        train_val_network.MergeFrom(hidden_layers)
+        for layer in hidden_layers:
+            # don't add layers with names starting with "deploy_" to train/val network
+            if not layer.name.startswith('deploy_'):
+                train_val_network.layer.add().CopyFrom(layer)
 
```

Comment:
In the `generic` path, I'm stripping the prefix from the final layer name:
https://github.com/NVIDIA/DIGITS/blob/v3.0.0/digits/model/tasks/caffe_train.py#L556

Can we do the same here?